support/util/imagefetch.go

```diff
@@ -0,0 +1,63 @@
+package util
+
+import (
+	"context"
+	"fmt"
+	"github.com/blang/semver/v4"
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"github.com/openshift/hypershift/support/releaseinfo"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+// GetHypershiftComponentImage resolves the appropriate control plane operator
```

Comment:
>// GetHypershiftComponentImage resolves the appropriate control plane operator
// image

So Rename GetHypershiftComponentImage -> GetCPOImage?

Reply:
I should change the comment it's really not control-plane-operator the image holds multiple operator images in one

Reply:
comment changed